src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -574,6 +574,10 @@ Color.white, new Procedure("condition4"),
 					getRandomItem(random, GUIComponent.AnchorPoint.values())));
 			components.add(new Image(22, 31, "picture2", false, new Procedure("condition2"),
 					getRandomItem(random, GUIComponent.AnchorPoint.values())));
+			components.add(new Sprite(25, 30, "sprite1", 16, 16, new Procedure("condition1"),
+					new Procedure("number1"), getRandomItem(random, GUIComponent.AnchorPoint.values())));
+			components.add(new Sprite(30, 35, "sprite2", 32, 16, new Procedure("condition2"),

```

Comment:
If the condition procedure is optional, also test a case where the condition is not present, example:

![image](https://github.com/user-attachments/assets/53b3f482-e8c7-4a45-b6ac-ec1473c2addf)